tests/test_cli.py

```diff
@@ -95,3 +95,10 @@ def test_pyproject_toml_resolved(
 def test_pyproject_toml_dir(tmp_path: Path) -> None:
     (tmp_path / "pyproject.toml").write_text("")
     cli_args([str(tmp_path)])
+
+
+def test_tox_failure(tmp_path: Path, monkeypatch: pytest.MonkeyPatch) -> None:
+    monkeypatch.syspath_prepend(tmp_path)
+    tox_bin = tmp_path / "tox"
+    with tox_bin.open("wb"):
+        cli_args(["tox.ini"])
```

Comment:
```
monkeypatch.syspath_prepend(tmp_path)
(tmp_path / "tox").write_text()
cli_args(["tox.ini"])
```

I think you also need to set the fle executable, otherwise wil not be picked up.